test/src/linux/snapTest.ts

```diff
@@ -288,9 +288,35 @@ test.ifDevOrLinuxCi(
     },
     effectiveOptionComputed: async ({ snap, args }) => {
       expect(snap).toMatchSnapshot()
-      expect(snap.compression).toEqual("xz")
+      expect(snap.compression).toBeUndefined()
```

Comment:
I don't think this test should be changing. This test "compression option" is making sure that compression is properly applied, so it shouldn't be undefined.

Reply:
I think the easy fix to retain this logic is to add to the config, `useTemplateApp: false`

Reply:
👍 I've made this change now